static/js/publisher/release/actions/releases.test.js

```diff
@@ -12,4 +29,234 @@ describe("releases actions", () => {
       expect(updateReleases(releases).payload.releases).toEqual(releases);
     });
   });
+
+  describe("handleCloseResponse", () => {
+    it("should dispatch CLOSE_CHANNEL_SUCCESS when successfull", () => {
+      const store = mockStore({});
+
+      handleCloseResponse(
```

Comment:
Now that I don't remember what these functions are doing is a nice time to review this tests for readability.

A bit of nit picking, but it would be nice to pass named variables instead of incline values to the function call.
At least for the API response object. It took me a while to understand what we are passing, why "edge" is mentioned twice etc.

Something like `handleCloseResponse(store.dispatch, apiResponse, ["latest/edge"])` would be a bit more understandable.